Dockerfile

```diff
@@ -0,0 +1,27 @@
+FROM node:boron
```

Comment:
We can use some light image here https://github.com/sickp/docker-alpine-node

Reply:
I think @AnshulMalik is correct. `node:boron` is 266 MB while `node:boron-alpine` is just 19MB. Using an `alpine` based image would be beneficial.

Reply:
Done @singhpratyush @AnshulMalik . Please review